### ISSUE TITLE
Uploading to device in background

### ISSUE THREAD

**Issue by pjnr1** [SKAARHOJ/skaarhoj-updater-releases#3]
Background Issue: 
      
```
Error: Object has been destroyed
    at BrowserWindow.setProgressBar (<anonymous>)
    at Bossa.<anonymous> (/Users/jenslindahl/repos/Skaarhoej/extra/SKAARHOJUpdaterBeta.app/Contents/Resources/app.asar/background.js:37:322050)
    at Bossa.emit (events.js:205:15)
```

---

SKAARHOJUpdaterBeta 0.9.18
Electron 6.1.4
darwin 18.7.0
Locale: da

MacOS 10.14.6

**Comment by s00500** [comment]
how did you manage to trigger that ?

**Comment by s00500** [comment]
Ok I think I found why that could occur, should be fixed in next release v0.9.19

**Comment by s00500** [comment]
see #4 event though the issue should have been fixed it seems like it did not...
